lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -426,6 +445,10 @@ def json_error(raw_response)
       def non_fractional_currency?(currency)
         CURRENCIES_WITHOUT_FRACTIONS.include?(currency.to_s)
       end
+
+      def emv_payment?(payment)
```

Comment:
Yeah - that way as EMV support gets added to other gateways (that only support credit cards getting passed in), they can just ask the credit card if it needs to be handled through EMV flows.